src/core/default_appearance.js

```diff
@@ -345,12 +317,13 @@ endcmap CMapName currentdict /CMap defineresource pop end end`;
 
   get baseFontRef() {
     const baseFont = new Dict(this.xref);
+    const identityH = Name.get("Identity-H");
     baseFont.set("BaseFont", this.fontName);
     baseFont.set("Type", Name.get("Font"));
     baseFont.set("Subtype", Name.get("Type0"));
-    baseFont.set("Encoding", Name.get("Identity-H"));
+    baseFont.set("Encoding", identityH);
```

Comment:
Given the [existing `Name` caching](https://github.com/mozilla/pdf.js/blob/49b2d9b5aff9d598323e9de80f8a545a8e65b4ea/src/core/primitives.js#L42-L45) we probably don't really need the temporary variable, since two back-to-back `Name.get("Identity-H")`-calls will return the *same exact object* anyway.
```suggestion
    baseFont.set("Encoding", Name.get("Identity-H"));
```